datacenter/ucp/2.0/guides/installation/upgrade.md

```diff
@@ -78,7 +78,7 @@ that you've installed using that node's root CA material.
 
     ```bash
     $ docker run --rm -it \
-      --name ucp \
+      --name ucp --privileged\
```

Comment:
@alexmavr can you confirm that the UCP installer needs to run in privileged mode? 🤔